REDEME.md

```diff
@@ -0,0 +1,12 @@
+#Hello World Project Documentation
+
+    this project consist to set-up linters in github and create basic HTML page and style it!
+
+## Built With
+
+- HTML and CSS
+
+
+## Authors
+
+- GitHub: [@nmeziany]
```

Comment:
- [ ] Please, consider that your README file name is incorrect - it is "REDEME". After fixing it, please, consider making it more informative - you can refer MICROVERSE template

Reply:
Modification for README file done and correction of its name too!